### PR TITLE
Fix some of the wording in the RHEV support documentation

### PR DESCRIPTION
**Pull request by whowutwut**
Fixed some details in the RHEV support after running some verification testing.

### CODE REVIEW

**Comment by whowutwut** [comment]
Found some environment issues was preventing the VM from launching, but the documentation is OK now for review @gurevichmark

File: docs/source/guides/admin-guides/manage_clusters/ppc64le/virtual_machines/RHEVHypervisor.rst [hunk 1]
```diff
@@ -1,7 +1,14 @@
 
-   At the time of this writing there is no ISO image availabe for RHEV. Individual RPM packages need to be downloaded.
+   Red Hat Virtualization (formally known as RHEV or Red Hat Enterprise Virtualization) is a virtualization solution provided by Red Hat.  
 
-   * Download *Management-Agent-Power-7* and *Power_Tools-7* RPMs from RedHat to the xCAT management node. Steps below assume all RPMs were downloaded to ``/install/post/otherpkgs/rhels7.3/ppc64le/RHEV4/4.0-GA``
+   At the time of this writing there is no RHEV-H prebuilt hypervisor image on Power LE.  The method for creating a Red Hat Hypervisor on Power LE is to install first install RHEL and apply the KVM support on top with the provided RPMs.
```
Comment (gurevichmark) [inline docs/source/guides/admin-guides/manage_clusters/ppc64le/virtual_machines/RHEVHypervisor.rst]: The "install first install" has one too many "install"

File: docs/source/guides/admin-guides/manage_clusters/ppc64le/virtual_machines/RHEVHypervisor.rst [hunk 2]
```diff
@@ -1,7 +1,14 @@
...
+   Red Hat Virtualization (formally known as RHEV or Red Hat Enterprise Virtualization) is a virtualization solution provided by Red Hat.  
 
-   * Download *Management-Agent-Power-7* and *Power_Tools-7* RPMs from RedHat to the xCAT management node. Steps below assume all RPMs were downloaded to ``/install/post/otherpkgs/rhels7.3/ppc64le/RHEV4/4.0-GA``
+   At the time of this writing there is no RHEV-H prebuilt hypervisor image on Power LE.  The method for creating a Red Hat Hypervisor on Power LE is to install first install RHEL and apply the KVM support on top with the provided RPMs. 
+
+   Obtain and download the RHEV RPM packages from the Red Hat download site.  
+
+       * Management-Agent-Power-7
+       * Power_Tools-7 
+
```
Comment (gurevichmark) [inline docs/source/guides/admin-guides/manage_clusters/ppc64le/virtual_machines/RHEVHypervisor.rst]: Do we want to mention that the packages are needed from 
`http://people.redhat.com/~thuth/qemu-2.6-28/`, or just say version 2.6 of qemu packages is required at a minimum ?

Comment (whowutwut) [inline docs/source/guides/admin-guides/manage_clusters/ppc64le/virtual_machines/RHEVHypervisor.rst]: Yes, we should mention that in the release notes.   That seems to be a workaround they provided

**Pull request merged**